src/widgetastic_patternfly4/ouia.py

```diff
@@ -0,0 +1,45 @@
+import importlib
+import sys
+
+from widgetastic.utils import ParametrizedLocator
+
+
+class OUIAMixin:
+
+    ROOT = ParametrizedLocator(
+        './/*[@data-ouia-component-type="PF4/{@component_type}" and '
```

Comment:
i believe we should still match the tag name for the root element,

Reply:
What's the point? Different components have different root tags. We should parametrize then this part.

Reply:
Another point is not to rely on any of html tag because it can be changed unlike OUIA attr.